wallet/structs.go

```diff
@@ -1017,6 +1017,32 @@ type RequestCreateWallet struct {
 	Language string `json:"language"`
 }
 
+// GenerateFromKeys()
+type RequestGenerateFromKeys struct {
+	// (Optional) The block height to restore the wallet from. (Defaults to 0)
+	RestoreHeight int64 `json:"restore_height"`
+	// The wallet's file name on the RPC server.
+	Filename string `json:"filename"`
+	// The wallet's primary address.
+	Address string `json:"address"`
+	// (Optional - omit to create a view-only wallet) The wallet's private spend key.
+	SpendKey string `json:"spendkey"`
+	// The wallet's private view key.
+	ViewKey string `json:"viewkey"`
+	// The wallet's password.
+	Password string `json:"password"`
+	// (Optional) If true, save the current wallet before generating the new wallet. (Defaults to true)
+	AutoSaveCurrent bool `json:"autosave_current"`
+}
```

Comment:
About 2 years ago this struct got an additional member to set the seed language of the new wallet:

    std::string language;

Not sure you leave out that on purpose or you worked with some documentation that does not yet contain it.

Reply:
Thank you very much! Indeed, this parameter is not yet specified in the wallet-rpc documentation. I have made the necessary corrections.